galsim/utilities.py

```diff
@@ -1145,6 +1145,8 @@ def combine_wave_list(*args):
             red_limit = min(red_limit, obj.red_limit)
         wave_list = np.union1d(wave_list, obj.wave_list)
     wave_list = wave_list[(wave_list >= blue_limit) & (wave_list <= red_limit)]
+    if blue_limit > red_limit:
```

Comment:
Should this maybe check if wave_list is empty?  Or can that only happen if blue_limit > red_limit?  

Also, even only having a single element is likely a problem, so might be worth giving an error for that too.

Reply:
Actually, I did try `if len(wave_list) == 0` before settling on `if blue_limit > red_limit`.  The former led to a dozen or so errors in the unit tests though.  Empty wave_lists are valid if you have an SED or Bandpass with an analytic wavelength dependence instead of one based off a LookupTable, so I don't think that's the right thing to check for.

I agree a single element would be suspicious.  I think the right way to handle that may be to change `>` -> `>=` above, rather than looking at wave_list directly.